sdk/appconfiguration/Azure.Data.AppConfiguration/tests/SessionRecords/ConfigurationLiveTests/AddExistingSetting.json

```diff
@@ -1,153 +0,0 @@
-{
-  "Entries": [
-    {
-      "RequestUri": "https://shrejaappconfiguration-azconfig-net.azconfig.io/kv/key-1022856440?label=test_label\u0026api-version=1.0",
-      "RequestMethod": "PUT",
-      "RequestHeaders": {
```

Comment:
I notice that the test recordings are removed from this repo, and I'm curious why there is no pull request or inclusion of these test recordings in the [Azure SDK Assets repo](https://github.com/Azure/azure-sdk-assets)?

Reply:
Recordings are pushed directly without the need for approval, so there's no pull request. Also, they're not pushed to a branch. Instead, the commit they are associated with is referenced via a [git tag](https://docs.github.com/en/rest/git/tags?apiVersion=2022-11-28).

Here's an example: [AppConfig tag](https://github.com/Azure/azure-sdk-assets/releases/tag/net%2Fappconfiguration%2FAzure.Data.AppConfiguration_2f82b09ab4) and its [associated commit](https://github.com/Azure/azure-sdk-assets/commit/2f82b09ab46bb8be00b544ac1f3ada211f3f577e).

Reply:
Ahhh, thanks for the clarification!